package.json

```diff
@@ -81,6 +87,8 @@
     "supertest": "^6.3.3",
     "ts-jest": "^29.0.5",
     "ts-node": "^10.9.1",
-    "tsconfig-paths-jest": "0.0.1"
+    "tsconfig-paths-jest": "0.0.1",
+    "ttypescript": "^1.5.15",
```

Comment:
This is like typescript, but allows us to use plugins like `typescript-transform-paths` below